### ISSUE TITLE
NotoSerifTibetan Some characters are not drawn correctly in MS-WORD (2010)

### ISSUE THREAD

**Issue by KazunariTsuboi** [googlefonts/noto-fonts#1539]
### Font 
NotoSerifTibetan-Regular.ttf

### Where the font came from, and when
https://github.com/googlefonts/noto-fonts/blob/master/phaseIII_only/unhinted/ttf/NotoSerifTibetan/NotoSerifTibetan-Regular.ttf
Date: 2019-06-13

### Font Version
Sans: Version 2.001

### OS name and version
Microsoft Windows [Version 10.0.17134.829]

### Application name and version
Microsoft Word 2010 (Version 14.0.7232.5000)

### Issue
The following string is not drawn correctly in MS-Word (2010).
ཟཱ U+0F5F U+0F71
ངྨ U+0F44 U+0FA8
ངྨཱ U+0F44 U+0FA8 U+0F71
ངྨེ  U+0F44 U+0FA8 U+0F7A
ཆཱོ  U+0F46 U+0F71 U+0F7C

![image](https://user-images.githubusercontent.com/21096794/60309211-a2a70400-9987-11e9-9c46-39cc55abfcb8.png)

The drawing results by HarfBuzz are as expected.
![image](https://user-images.githubusercontent.com/21096794/60309237-c702e080-9987-11e9-99ea-84d2767ea134.png)

**Comment by marekjez86** [comment]
@KazunariTsuboi : my "gut feeling" is that Windows 10 doesn't know the language is Tibetan (is there a way to set this?) or if it knows doesn't know how to render it, but I could be wrong

**Comment by dscorbett** [comment]
Microsoft Word 2010 is rather old; is it possible that it is using [the old Tibetan shaper](https://docs.microsoft.com/en-us/typography/script-development/tibetan) instead of USE? The Tibetan shaper enables 'abvm' and 'blwm' but this font uses 'mark'.

**Comment by KazunariTsuboi** [comment]
I displayed NotoSerifTibetan on Word in another environment.
>Windows 10 [10.0.17134.829]
>Word 2013 (Version 15.0.5127.1000)

In this environment it was drawn correctly.
![image](https://user-images.githubusercontent.com/21096794/60404203-88686280-9be1-11e9-8e5e-858b5a11d011.png)

Maybe, Word 2010 uses the old Tibetan shaper, and the new Word looks like using USE.

**Comment by KazunariTsuboi** [comment]
I'm sorry, I was confusing GSUB and GPOS. Certainly, mark feature is used in NotoSerifTibetan GPOS. Word 2010 seems to use the old Tibetan shaper.

**Comment by punchcutter** [comment]
@KazunariTsuboi Yes, it's on the website at https://docs.microsoft.com/en-us/typography/script-development/tibetan where it says 

> Starting in Windows 10, Tibetan will be supported by the Universal Shaping Engine rather than a stand-alone shaping engine

**Comment by KazunariTsuboi** [comment]
I tried to make a simple prototype.
It is NotoSerifTibetan which used 'abvm' and 'blwm' instead of 'mark'.
The display on MS-WORD (2010) is now successful. I have attached a prototype.
![image](https://user-images.githubusercontent.com/21096794/60488655-74a22680-9cdd-11e9-9109-fc578a4c1bad.png)

The generation method is as follows.
Added "Tibt" to indicScripts in MarkFeatureWriter Class of ufo2ft/featureWriters/markFeatureWriter.py
Then I used 'fontmake' to generate from NotoSerifTibetan-MM.glyphs.

I tested Tibetan on noto-source using fontdiff.
Attach the resulting PDF.
Apparently, it looks like it is going well.

I do not know all the backgrounds that 'mark' is used for Tibetan but not 'abvm' and 'blwm'. 
I just report it because it was an interesting result.
[TibetanTEST.zip](https://github.com/googlefonts/noto-fonts/files/3348563/TibetanTEST.zip)
